hummingbird/ml/operator_converters/_binarizer_implementations.py

```diff
@@ -0,0 +1,27 @@
+# -------------------------------------------------------------------------
```

Comment:
Usually we put into `_*_implementations.py` all the implementations shared between onnx and pytorch. In this case we don't have an onnx implementation, so I think we can just have the implementation into the `binarizer.py` file.

Reply:
I considered this, but do we have an example currently that is just in a .py?  Not for SKL.  Maybe a few for onnx?

Reply:
Yes. Like `sv` and `iforest`.